go.mod

```diff
@@ -5,7 +5,6 @@ go 1.13
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/evanphx/json-patch v4.5.0+incompatible
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
Would expect to see some go.sum update as well, have you run `make vendor`?

Reply:
Let me check. Sometimes I need to remove `go.sum` so the vendor takes effect.

Reply:
> Sometimes I need to remove go.sum so the vendor takes effect.

You should never need to do that... What errors are you getting that make you want to do that?